### DISCUSSION TITLE
更新后webdav无法使用

### DISCUSSION THREAD

**Discussion by 6396036** [AlistGo/alist#890] [Q&A] [answered]
我的配置情况：
A机安装alist （国外大盘鸡，对本地网络不友好）；
B机安装alist（对本地网络好）；
我的使用情况：
B机通过alist挂载A机，在B机添加时选择“webdav proxy”，实现大盘鸡的快速访问。

2.1之前版本一直使用正常，非常感谢这么好的软件直接搞定我的要害，非常爽。
可是自从昨天手动更新后，webdav就无法使用了，一直显示Status 404。
重装、换token，都试了，一直不行。
单独访问A、B机都是正常的。

求大佬指导如何解决，o(╥﹏╥)o

**Comment by xhofe** [comment, marked as answer]
是否最新的2.3.2？不是的话更新就行了。

**Reply by 6396036** [reply]
更新到v2.3.2后问题解决，非常感谢大佬，创造了这么优秀的软件！
